tools/operator/launcher/networkspec.yaml

```diff
@@ -2,11 +2,35 @@
 #!
 #! SPDX-License-Identifier: Apache-2.0
 ---
-#! fabricVersion:
-#! Released images are pulled from docker hub hyperledger/, e.g. 1.4.5 or 2.0.0
-#! Development stream images are pulled from
-#! hyperledger-fabric.jfrog.io/, e.g. 1.4.5-stable or 2.0.0-stable
-fabricVersion: 1.4.5-stable
+#! dockerOrg and dockerTag specify the organization name and image tag
+#! when pulling docker images. For example:
+#!
+#! dockerOrg: hyperledger-fabric.jfrog.io
+#! dockerTag: amd64-2.0-stable
```

Comment:
shall this be 2.1 instead of 2.0?

Reply:
I actually considered it while doing this, but decided I didn't really care. Its just an example and I don't really plan to increment it at all ever.